test/NuGet.Core.Tests/NuGet.PackageManagement.Test/NuGet.PackageManagement.Test.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>$(NETFXTargetFramework)</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(IsXPlat)' == 'true' ">$(NETCoreTargetFramework)</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(IsXPlat)' == 'true' ">$(NETCoreTargetFramework);netcoreapp3.0</TargetFrameworks>
```

Comment:
Update the property in the cmmon.targets

This also seems unnecessary. 
Seems like dead code, as earlier it should have failed because netcoreapp2.1 is not compatible with net472

Reply:
Thanks! Changed accordingly.